inline_verifier.go

```diff
@@ -0,0 +1,223 @@
+package ghostferry
+
+import (
+	"bytes"
+	"database/sql"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/golang/snappy"
+	"github.com/sirupsen/logrus"
+)
+
+type InlineVerifier struct {
+	SourceDB *sql.DB
+	TargetDB *sql.DB
+
+	sourceStmtCache *StmtCache
+	targetStmtCache *StmtCache
+	logger          *logrus.Entry
+}
+
+func (v *InlineVerifier) VerifyBeforeCutover() error {
+	// TODO: Iterate until the reverify queue is small enough
+	return nil
+}
+
+func (v *InlineVerifier) VerifyDuringCutover() (VerificationResult, error) {
+	// TODO: verify everything within the reverify queue.
+	return VerificationResult{}, nil
+}
+
+func (v *InlineVerifier) StartInBackground() error {
+	// not needed?
```

Comment:
These methods (`StartInBackground, Wait, Result`) are to satisfy the `Verifier` interface. They might be needed to play nice with the ControlServer interface for later.